tools/depends/target/waylandpp/build-static.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt	2023-03-21 16:58:24.080908279 +0100
++++ b/CMakeLists.txt	2023-03-21 16:58:07.680958170 +0100
+@@ -139,7 +139,7 @@
+
+   # library building helper functions
+   function(define_library TARGET CFLAGS LIBRARIES HEADERS)
+-    add_library("${TARGET}" ${ARGN})
++    add_library("${TARGET}" STATIC ${ARGN})
```

Comment:
Can't we just use the cmake option `-D BUILD_SHARED_LIBS=OFF`?

https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html

Reply:
right, that's much better, updated